src/QtDBMigration.cpp

```diff
@@ -84,13 +91,20 @@ int QtDBMigrationPrivate::version()
 
 bool QtDBMigrationPrivate::migrate()
 {
-    if (m_config->schemaVersions().isEmpty()) {
```

Comment:
Do not change this. As you can see, you just duplicated code. Made the abstraction (AbstractConfig) completely useless.

Reply:
Changes reverted.